packages/zoe/src/contracts/barterExchange.js

```diff
@@ -0,0 +1,188 @@
+// @ts-check
+
```

Comment:
Would it be possible to have a very simple example, like atomicSwap or coveredCall changed to allow for different user keywords? I was getting bogged down in this one. Also, could we see how multipoolAutoswap would change?